doc/api/assert.md

```diff
@@ -469,7 +469,7 @@ parameter is an instance of an [`Error`][] then it will be thrown instead of the
 <!-- YAML
 added: v0.1.21
 -->
-* `message` {any} **Default:** `'Failed'`
+* `message` {any} **Default:** `'Failed'`.
```

Comment:
I'll stop after this one, but basically I don't think that we should put periods after values that aren't otherwoise part of a sentence or even a phrase.

Reply:
I've removed the periods if:
* default value statement is not a sentence;
* default value statement is not after a sentence (after a dot).